hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestFileSystemViewCommand.java

```diff
@@ -59,24 +59,73 @@
 @Tag("functional")
 public class TestFileSystemViewCommand extends CLIFunctionalTestHarness {
 
+  private String unpartitionedTablePath;
+  private String partitionedTablePath;
   private String partitionPath;
-  private SyncableFileSystemView fsView;
+  private SyncableFileSystemView unpartitionedFsView;
+  private SyncableFileSystemView partitionedFsView;
 
   @BeforeEach
   public void init() throws IOException {
+    createUnpartitionedTable();
+    createPartitionedTable();
+  }
+
+  private void createUnpartitionedTable() throws IOException {
     HoodieCLI.conf = hadoopConf();
 
     // Create table and connect
-    String tableName = tableName();
-    String tablePath = tablePath(tableName);
+    String unpartitionedTableName = "unpartitioned_" + tableName();
+    unpartitionedTablePath = tablePath(unpartitionedTableName);
```

Comment:
nit: use `non-partitioned` (`NonPartitioned`) instead of `unpartitioned`, in all appearances

Reply:
done